### DISCUSSION TITLE
New upload category: Upscale models

### DISCUSSION THREAD

**Discussion by Nenotriple** [civitai/civitai#307] [Ideas]
The Upscale wiki https://upscale.wiki/wiki/Model_Database is a great resource, but I would love to see people sharing their upscale models on Civitai. 

Along with a new upload category that handles these models, a new image viewer that that's designed for comparisons would be very handy. Something like what's available on https://imgsli.com/ would be ideal in my opinion.

**Comment by WASasquatch** [comment]
Imo, there aren't enough releasing models for this to be a viable community aspect more than it would just add to cost by being a bandwidth leech. That upscale list has only moderately changed in a few years, also these often have limitations that make their use questionable, like codeformer non-commercial becuase of CelebsA dataset, etc. 

Also a lot of these upscale models are really just garbage, least imo.  Like some of them are like "better than ESRGAN" then gives you horrible noodle line results and totally destroyed surface details from over denoising.
